src/main/java/com/google/ehub/data/EntertainmentItemDatastore.java

```diff
@@ -84,21 +128,24 @@ public static void addEntertainmentItemToDatastore(
                         title.toLowerCase())))));
   }
 
-  private static List<EntertainmentItem> getListFromQuery(Query query) {
+  private List<EntertainmentItem> getListFromQuery(Query query) {
     PreparedQuery queryResults = datastoreService.prepare(query);
 
     List<EntertainmentItem> entertainmentItemList = new ArrayList<>();
 
     for (Entity entertainmentItemEntity : queryResults.asIterable()) {
-      String title = (String) entertainmentItemEntity.getProperty(TITLE_PROPERTY_KEY);
-      String description = (String) entertainmentItemEntity.getProperty(DESCRIPTION_PROPERTY_KEY);
-      String imageURL = (String) entertainmentItemEntity.getProperty(IMAGE_URL_PROPERTY_KEY);
-
-      entertainmentItemList.add(new EntertainmentItem(title, description, imageURL));
+      entertainmentItemList.add(getItemFromEntity(entertainmentItemEntity));
     }
 
     return entertainmentItemList;
   }
 
-  private EntertainmentItemDatastore() {}
+  private EntertainmentItem getItemFromEntity(Entity entertainmentItemEntity) {
```

Comment:
nit: "create"

"get" sort of implies you are fetching it from somewhere

Reply:
I changed the name for the methods that use "get" into "create" to indicate that they are not retrieved from somewhere but instead they are being created.